pkg/api/controller.go

```diff
@@ -372,6 +372,10 @@ func (c *Controller) StartBackgroundTasks(reloadCtx context.Context) {
 		ext.EnableMetricsExtension(c.Config, c.Log, c.Config.Storage.RootDirectory)
 		ext.EnableSearchExtension(c.Config, c.StoreController, c.MetaDB, taskScheduler, c.CveScanner, c.Log)
 	}
+	// runs once if metrics are enabled
+	if c.Config.IsMetricsEnabled() {
+		c.StoreController.DefaultStore.PopulateStorageMetrics(time.Duration(0), taskScheduler)
```

Comment:
here we run storage metric for both local and s3, but getDirSize() func supports only local imagestore.

Either update that function to take the storage/types.go.Driver implementation as an arg and use the driver to get dir size, or run it only for local storage, and we fixed this later.

otherwise lgtm

Reply:
Try to fix this now so we are done with this.